Source/Core/Core/PowerPC/JitArm64/Jit.h

```diff
@@ -311,6 +311,9 @@ class JitArm64 : public JitBase, public Arm64Gen::ARM64CodeBlock, public CommonA
   void BeginTimeProfile(JitBlock* b);
   void EndTimeProfile(JitBlock* b);
 
+  // RMEM
```

Comment:
There is nothing called "RMEM" in JitArm64. I assume you mean MEM_REG. Though with that said, I'm not sure how much information this comment really imparts...